src/views/ModelPlan/TaskList/ITSolutions/SolutionImplementation/index.tsx

```diff
@@ -167,14 +171,26 @@ const SolutionImplementation = () => {
                 </span>
               </Alert>
             );
-
             history.push(`/models/${modelID}/task-list/it-solutions`);
             // Go back but still save solution details
           } else if (redirect === 'back') {
             history.push(
               `/models/${modelID}/task-list/it-solutions/${operationalNeedID}/select-solutions`
             );
 
+            // If isUpdatingStatus, go to previous page, otherwise, go to tracker
+          } else if (isUpdatingStatus) {
+            showMessageOnNextPage(
+              <Alert type="success" slim className="margin-y-4">
+                <span className="mandatory-fields-alert__text">
+                  {t('successStatusUpdated', {
+                    operationalNeedName: operationalNeed.name
+                  })}
+                </span>
+              </Alert>
+            );
+            history.goBack();
```

Comment:
@patrickseguraoddball, I made the conditional check and routing more verbose in order to incorporate a different success message when updating the status.